lib/fluent/plugin/filter_prometheus.rb

```diff
@@ -17,7 +18,7 @@ def multi_workers_ready?
 
     def configure(conf)
       super
-      labels = Fluent::Plugin::Prometheus.parse_labels_elements(conf)
```

Comment:
if fluentd is multiple worker mode, this plugin always adds `worker_id` label even if users request to not aggregated path.

Reply:
Yes, as discussed in https://github.com/fluent/fluent-plugin-prometheus/pull/109#discussion_r320854908

Reply:
ah, now we populate worker_id as opt-in by specifying  a label explicitly. It becomes to populate worker_id by default with this change.
I want to avoid implicitly adding extra labels only for the aggregated path and also avoid emitting the exactly same metrics after merging, so I prefer to add worker_id as a label by default.